### PR TITLE
Modal confirmation delete done

### PR DESCRIPTION
**Pull request by maximmft**

### CODE REVIEW

**Review by Coline1186 — APPROVED** [review]

File: client/src/components/Modal/ModalConf/ModalConf.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/* MOBILE */
+#modal-conf-profil {
```
Comment (VinchC) [inline client/src/components/Modal/ModalConf/ModalConf.css]: profile

File: client/src/components/Modal/ModalConf/ModalConf.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  right: 17%;
+  position: absolute;
+  z-index: 99;
+  top: 45%;
+}
+
+#modal-conf-profil h1 {
```
Comment (VinchC) [inline client/src/components/Modal/ModalConf/ModalConf.css]: profile

File: client/src/components/Modal/ModalConf/ModalConf.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+.buttons-conf-modal button {
+  padding: 10px 30px;
+  margin: 0 5px;
+  background-color: white;
+  color: black;
+}
+
+/* DESKTOP */
+@media screen and (min-width: 1060px) {
+  #modal-conf-profil {
```
Comment (VinchC) [inline client/src/components/Modal/ModalConf/ModalConf.css]: profile

File: client/src/components/Modal/ModalConf/ModalConf.css [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    left: 25;
+    top: 35%;
+  }
+
+  #modal-conf-profil h1 {
```
Comment (VinchC) [inline client/src/components/Modal/ModalConf/ModalConf.css]: profile

File: client/src/components/Modal/ModalConf/ModalConf.jsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import PropTypes from "prop-types";
+import "./ModalConf.css";
+
+function ModalConf({ setModalConfOpen, handleDelete }) {
+    const handleCloseModal = () => {
+      setModalConfOpen(false);
+    };
+
+  return (
+      <div id="modal-conf-profil">
```
Comment (VinchC) [inline client/src/components/Modal/ModalConf/ModalConf.jsx]: profile

File: client/src/assets/images/illustrations/flower4 - Copie.png
Comment (VinchC) [inline client/src/assets/images/illustrations/flower4 - Copie.png:1]: delete the - Copie in the file name

File: client/src/router.jsx [hunk 6]
```diff
@@ -35,7 +35,10 @@ const router = createBrowserRouter([
       },
       { path: "/sellingPage", element: <SellingPage />,
        },
-      { path: "/profilePage", element: <ProfilePage /> },
+      { path: "/profilePage/", element: <ProfilePage />,
```
Comment (VinchC) [inline client/src/router.jsx]: rename in /profile without Page

File: client/src/router.jsx [hunk 7]
```diff
@@ -35,7 +35,10 @@ const router = createBrowserRouter([
       },
       { path: "/sellingPage", element: <SellingPage />,
```
Comment (VinchC) [inline client/src/router.jsx]: rename in /selling without Page

**Review by VinchC — CHANGES_REQUESTED** [review]

**Review by VinchC — APPROVED** [review]

**Pull request merged**